flow360/accounts_utils.py

```diff
@@ -58,6 +58,8 @@ def _get_supported_users():
         try:
             response = http.portal_api_get("auth/credential")
             supported_users = response["guestUsers"]
+            if supported_users is None:
```

Comment:
if supported_users:
            	for entry in supported_users:
                    entry["userEmail"] = entry.pop("email")
                    entry["userIdentity"] = entry.pop("identity")
                return supported_users